packages/actionmenu/src/react/index.js

```diff
@@ -193,104 +196,106 @@ const calcNestedMenuPosition = (menuWidth, origin) =>
     }
   }[origin])
 
-class Item extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      isNestedRendered: false
+function usePrevious(value) {
+  const ref = React.useRef()
+
+  React.useEffect(() => {
+    ref.current = value
+  }, [value])
+
+  return ref.current
+}
+
+const Item = props => {
+  const { icon, isActive, ...rest } = props
+  const Tag = props.href ? 'a' : 'button'
```

Comment:
"Tag" was a confusing name for me.  For instance, we have a DS "Tag".  This was "tagName" in the previous solution, which I understand, but that looks odd: `<TagName />`.  Maybe semantically what it is? Which is what?  Root/Container/etc.??  What do you think?

Reply:
yeah good call on not using `Tag`. `TagName` could work, i've also seen `Element` in the wild